src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/CreateGoalTest.kt

```diff
@@ -155,5 +152,9 @@ class CreateGoalTest : IntegrationTestBase() {
     val actual = actionPlanRepository.findByPrisonNumber(prisonNumber)
     assertThat(actual).isForPrisonNumber(prisonNumber)
     assertThat(actual).hasNumberOfGoals(2)
+    val goal = actual!!.goals!![1]
+    assertThat(goal).hasTitle(createRequest.title)
```

Comment:
Might be worth chaining these, and creating an assertion method for notes?
```
assertThat(goal)
    .hasTitle(createRequest.title)
    .hasNumberOfSteps(createRequest.steps.size)
    .doesNotHaveNotes()
```

Reply:
Sure, but I'll do it in my next PR 👍